File: src/cxx/VulkanContext/Exceptions/NotSupportedExtensionException.cpp
//
// Created by kgaft on 7/26/23.
//

#include "NotSupportedExtensionException.h"

NotSupportedExtensionException::NotSupportedExtensionException(const char *message) : message(message){

}

const char *NotSupportedExtensionException::what() {
    return message;
}

File: src/main/java/com/kgaft/VulkanContext/Exceptions/NotSupportedExtensionException.java
package com.kgaft.VulkanContext.Exceptions;

public class NotSupportedExtensionException extends Throwable {
    public NotSupportedExtensionException(String message) {
        this.message = message;
    }
    private String message;
    @Override
    public String getMessage() {
        return message;
    }
}
